Sources/NavigationStack/NavigationStack.swift

```diff
@@ -45,14 +45,19 @@ public enum PopDestination {
 // MARK: ViewModel
 
 public class NavigationStack: ObservableObject {
+
+    /// Default transition animation
+    public static let defaultAnimation = Animation.easeOut(duration: 0.2)
```

Comment:
I'd rename this to `defaultEasing`, since `animation` might be confusing respect to another thing we have here which is `transition`.
```suggestion
    public static let defaultEasing = Animation.easeOut(duration: 0.2)
```